internal/db/schema/migrations/oss/postgres/100/04_resource.up.sql

```diff
@@ -0,0 +1,71 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+  create table resource_enm (
```

Comment:
Can we add rename this to something like `iam_resource_enm` or `iam_grant_resource_enm`? 
I'm not sure of the exact name but I think it at least needs the `iam_` prefix since that is the subsytem/package that it most belongs with. @tmessi thoughts?